pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -575,6 +586,25 @@ func (p *staticPolicy) calculateHints(machineState state.NUMANodeMap, pod *v1.Po
 	return hints
 }
 
+func (p *staticPolicy) selectableNUMANodes(machineState state.NUMANodeMap) []int {
+	numaNodes := make([]int, 0, len(machineState))
+	for nodeID := range machineState {
+		if !p.nodeHasCPUs(nodeID) {
+			continue
+		}
+		numaNodes = append(numaNodes, nodeID)
+	}
+	sort.Ints(numaNodes)
+	return numaNodes
+}
+
+func (p *staticPolicy) nodeHasCPUs(nodeID int) bool {
+	if len(p.numaNodesWithCPUs) == 0 {
+		return true
+	}
```

Comment:
I don't think we should do this. If there is some weird esoteric hardware that would require this, then the `TopologyManager` should not be used. We can advise them to just disable it. We should add a follow.up to improve the docs around this.